perl6.html.markdown

```diff
@@ -193,6 +193,15 @@ sub mutate($n is rw) {
   say "\$n is now $n !";
 }
 
+my $m = 42;
+mutate $m; # $n is now 43 !
```

Comment:
I put it there to show how passing a variable works with 'is rw' and on the next few lines I show how it doesn't work on passing an integer.  If you think this part is unnecessary I can remove it